questions/kart-course.md

```diff
@@ -0,0 +1,71 @@
+# Kart Kourse
+
+## Question
+
+Welcome to **Kart Kourse**! Here is the problem statement:
+
+You are given a list of racers' names in race position order and a list of
+position changes. Your task is to write a function that returns the new order of
+racers after applying the position changes.
+
+## Input
+
+Here is an example input:
+
+```python
+racers = ["Mario", "Luigi", "Peach", "Yoshi", "Toad"]
+position_changes = [2, 1, -1, 0, -2]
+```
+
+The first element in `position_changes` indicates that the racer in the second
+position (Luigi) advances two places. The second element indicates that the
+racer in the third position (Peach) advances one place. The third element
+indicates that the racer in the fourth position (Yoshi) moves down one place.
+The fourth element indicates that the racer in the fifth position (Toad) stays
+in place. The fifth element indicates that the racer in the first position
+(Mario) moves down two places.
+
```

Comment:
explaning `position_changes` could be clarified a little better. For example: `position_changes[i]` represents the position for `racers[i+1]`, and the last element in `position_changes` is for `racers[0]`